messages/doctor.js

```diff
@@ -28,5 +29,5 @@ Run CLI doctor diagnostics for a specific plugin:
   doctorNotInitializedError: 'Must first initialize a new SfDoctor.',
   doctorAlreadyInitializedError: 'SfDoctor has already been initialized.',
   pluginNotInstalledError:
-    'Specified plugin [%s] isn\'t installed. Install it, correct the name, or choose another plugin.',
+    "Specified plugin [%s] isn't installed. Install it, correct the name, or choose another plugin.",
```

Comment:
Doesn't this fail linting for using double quotes?

Reply:
not sure what was going on there, I'm not seeing any linting errors, reverting it